src/webview_provider/sqlLineagePanel.ts

```diff
@@ -309,7 +309,7 @@ export class SQLLineagePanel implements Disposable {
           status: true,
           body: {
             showSelectEdges: config.get("showSelectEdges", true),
-            showNonSelectEdges: config.get("showNonSelectEdges", true),
+            showNonSelectEdges: config.get("showNonSelectEdges", false),
```

Comment:
@AdiGajbhiye Why do we have it in SQLLineage panel?

Reply:
copied by mistake in earlier PRs. not getting used.

Reply:
@AdiGajbhiye should we remove it?

Reply:
or atleast add a todo for now